packages/react-components/react-table/src/components/TableRow/useTableRowStyles.ts

```diff
@@ -54,6 +56,21 @@ const useStyles = makeStyles({
   root: {
     color: tokens.colorNeutralForeground1,
     boxSizing: 'border-box',
+    ...createCustomFocusIndicatorStyle(
+      {
+        [`& .${tableSelectionCellClassNames.root}`]: {
```

Comment:
No way this can be applied on the `TableSelectionCell` component through some contextual flags? instead of styling it from the parent and using the `tableSelectionCellClassNames.root`

Reply:
@bsunderhus  I updated the implem with context and event handlers in https://github.com/microsoft/fluentui/pull/25202/commits/4072f13971ba79e8aca7a80011a57f9711715d5a

WDYT ?

Reply:
I ended up reverting it for the following reasons
* Every time context value changes the entire row needs to be re-rendered
* From a naive PoV I only change `onMouseEnter` `onMouseLeave` `onFocus` `onBlur` event handlers, then realized there were edge cases like mousedown event which will **blur** the current element and focus on body.

I think that css is definitely the 'stupidest' and most straightforward solution here 🙈

Reply:
Sounds great! thx for taking a look into it!